.ci/setup.sh

```diff
@@ -19,6 +19,46 @@ if [[ $OS_NAME == "macos" ]]; then
     fi
     wget -q -O conda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
 else  # Linux
+    if [[ $IN_UBUNTU_LATEST_CONTAINER == "true" ]]; then
+
+        # fixes error "unable to initialize frontend: Dialog"
+        # https://github.com/moby/moby/issues/27988#issuecomment-462809153
+        echo 'debconf debconf/frontend select Noninteractive' | sudo debconf-set-selections
+
+        sudo apt-get update || exit -1
+        sudo apt-get install -y --no-install-recommends \
+            software-properties-common || exit  -1
```

Comment:
I don't think this is something critical we should check exit code for. Perhaps it was left after debugging "`sudo` inside container" issue.
```suggestion
        sudo apt-get update
        sudo apt-get install -y --no-install-recommends \
            software-properties-common
```